package.json

```diff
@@ -22,16 +22,18 @@
     "use-callback-ref": "^1.0.1"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.4",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
+    "@babel/core": "^7.7.7",
+    "@babel/plugin-syntax-dynamic-import": "^7.7.4",
```

Comment:
Uninstalled, no need for this https://github.com/babel/babel/issues/9066#issuecomment-441040634